### ISSUE TITLE
No longer compatible with latest version of @npmcli/arborist 

### ISSUE THREAD

**Issue by Queatz** [tnobody/lerna-audit#25]
`Cannot find module '@npmcli/arborist/lib/update-root-package-json'`

**Comment by hiro5id** [comment]
I have fixed this in the pull request https://github.com/tnobody/lerna-audit/pull/26
@tnobody or @svettwer please accept my pull request

**Comment by scott-thrillist** [comment]
Yup, fresh install and getting this message:

```
% npx lerna-audit
Cannot find module '@npmcli/arborist/lib/update-root-package-json'
Require stack:
- /node_modules/lerna-audit/save-package-json.function.js
- /node_modules/lerna-audit/main.js
```

**Comment by hiro5id** [comment]
is anyone going to accept my pull request? or is there something wrong with it?

**Comment by svettwer** [comment]
> is anyone going to accept my pull request?

Most likely, yes. It's just vacation time, that's all. =)

**Comment by svettwer** [comment]
Solved in c8d3a9008874b6a6c07371acb6ce05a923c35432
